pkg/image/api/helper.go

```diff
@@ -33,8 +33,9 @@ const (
 	containerImageEntrypointAnnotationFormatKey = "openshift.io/container.%s.image.entrypoint"
 )
 
-// TODO remove (base, tag, id)
-func parseRepositoryTag(repos string) (string, string, string) {
+// ParseRepositoryTag splits a string into its name component and either tag or id if present.
+// TODO remove
+func parseRepositoryTag(repos string) (base string, tag string, id string) {
```

Comment:
Actually you could drop that already and use the next two functions `ParseImageStreamImageName` and `ParseImageStreamTagName`. That would remove the partial code duplication.